{{ cookiecutter.repo_name }}/inst/WORDLIST

```diff
@@ -0,0 +1,13 @@
+Acknowledgements
```

Comment:
**Further investigation**

When I create a new repository with R pre-commit hooks with no R files, set up `renv`, and run the hooks, I get the following error:

<details>
    <summary>Error in spell-check hook</summary>
    <pre>
The following spelling errors were found:
  WORD                   FOUND IN
AQA                    README.md:3
                       aqa_plan.md:3
                       README.md:54,56
BUILDDIR               Makefile:9,13,20,22,24
                       make.bat:11,28,32
Commonmark             README.md:21
ContainerShowSidebar   .DS_Store:4
                       .DS_Store:4
GDS                    pull_request_template.md:4
                       README.md:115
LazyData               DESCRIPTION:21
LinkingTo              settings.dcf:3
MAKEFILE               Makefile:32
Makefile               Makefile:15,19
NUL                    make.bat:15
NumPy                  README.md:14
PYTHONPATH             .envrc:12,13
ReStructuredText       README.md:9,13,27,30,34,50,55,64,77
                       README.md:42
Roxygen                DESCRIPTION:22
RoxygenNote            DESCRIPTION:23
SOURCEDIR              Makefile:8,13,20
                       make.bat:10,28,32
SPHINXBUILD            Makefile:7,13,20
                       make.bat:7,8,15,19,28,32
SPHINXOPTS             Makefile:6,13,18,20
                       make.bat:28,32
SRC                    .envrc:35,36,37,38,39,40
ShowPathbar            .DS_Store:4
                       .DS_Store:4
ShowSidebar            .DS_Store:4
                       .DS_Store:4
ShowStatusBar          .DS_Store:4
                       .DS_Store:4
ShowTabView            .DS_Store:4
                       .DS_Store:4
ShowToolbar            .DS_Store:4
                       .DS_Store:4
VISUALISATIONS         .envrc:39
WindowBounds           .DS_Store:4
                       .DS_Store:4
addopts                pytest.ini:2
api                    Makefile:22,25
aut                    DESCRIPTION:7
awk                    Makefile:34
codeblock              README.md:34
cre                    DESCRIPTION:7
docopt                 DESCRIPTION:13
docstring              .flake8:3
                       README.md:50
docstrings             README.md:12,44,46,64
doctest                pytest.ini:2,3
dp                     make.bat:3
drivendata             Makefile:15
env                    .envrc:10,16
environ                .envrc:1,6
errorlevel             make.bat:16
github                 Makefile:15
goto                   make.bat:13,29
gpl                    DESCRIPTION:10
http                   .flake8:2
                       make.bat:24
https                  Makefile:15
init                   Makefile:53
lintr                  DESCRIPTION:15
makefile               Makefile:1
mit                    DESCRIPTION:10
myz                    .DS_Store:4
                       .DS_Store:4
ncol                   Makefile:35,42,46
optionflags            pytest.ini:3
os                     .envrc:1,4,6
popd                   make.bat:35
printf                 Makefile:40,47,49,51
pushd                  make.bat:3
pwd                    .envrc:13,19,26,29,32,35,43
pycache                .flake8:5
pycqa                  .flake8:2
pytest                 pytest.ini:1
                       requirements.txt:4
recommonmark           requirements.txt:5
rf                     Makefile:24,25
sed                    Makefile:19
setaf                  Makefile:37
setuptools             Makefile:7
sgr                    Makefile:17,38
src                    .envrc:12,34,35
styler                 DESCRIPTION:17
summarise              README.md:3
tput                   Makefile:17,35,37,38
uname                  Makefile:53
untracked              .envrc:15
usethis                DESCRIPTION:18
vcs                    settings.dcf:6
venv                   .flake8:5
visualisations         .envrc:39
                       README.md:87
vv                     pytest.ini:2
All spelling errors found were copied to inst/WORDLIST assuming they were not spelling errors and will be ignored in the future. Please  review the above list and for each word that is an actual typo:
 - fix it in the source code.
 - remove it again manually from inst/WORDLIST to make sure it's not
   ignored in the future.
 Then, try committing again.
Error: Spell check failed
Execution halted
    </pre>
</details>

Steps to recreate error:

1. In terminal, checkout the latest version of this branch;
2. Navigate outside of the `govcookiecutter` folder;
3. With `cookiecutter` installed in a Python virtual environment, run the following command:
    ```shell
    cookiecutter govcookiecutter
    ```
4. Use any values for the cookiecutter prompt options, except select `Yes` for the `add_r_precommit_hooks` option;
5. Navigate into your newly-created project from Step 4;
6. Initialise Git with `git init`;
7. Open an R console;
   ```shell
   r
   ```
8. Run the `renv::install()` command, and then the `renv::restore()` command, selecting `y`;
9. Quit the R console without saving (option `n`);
    ```shell
    q()
   ```
10. Install the pre-commit hooks; and
      ```shell
      pre-commit install
      ```
11. Run pre-commit locally.
     ```shell
     pre-commit run --all-files
     ```

Reply:
I can't reproduce that error.  For me, `pre-commit run --all-files` skips every hook.

Reply:
Argh I forgot to add a step! 😳 

🤦 My bad! Added Step 10 above!

Reply:
Ah okay, haha, thanks.  So are you suggesting that the updated `inst/WORDLIST` be added to the cookiecutter?

Reply:
Presumably? I feel a few more files need to be added to the exclusion - e.g. `pytest.ini`

Reply:
This should be resolved by https://github.com/ukgovdatascience/govcookiecutter/pull/9/commits/77a2b9759239888539780a46be089ea2b1ec47d5 now.